src/pages/test-report/test-report.effects.ts

```diff
@@ -105,7 +105,8 @@ export class TestReportEffects {
 
             const actions: Action[] = [new testsActions.SetActivityCode(result)];
             if (!isEmpty(currentTest.activityCode) && currentTest.activityCode !== result) {
-              actions.push(new testsActions.TestOutcomeChanged());
+              const label = result === '1' ? 'fail to pass' : 'pass to fail';
```

Comment:
Any requirement to know when terminate was changed?

Reply:
That's what I was thinking about as well. But we are purely interested here whether it changed from fail to pass or pass to fail. Would be used to prevent fraud